FF1Lib/Hacks.cs

```diff
@@ -1044,7 +1044,7 @@ public void MonsterInABox(MT19337 rng, Flags flags)
 			PutInBank(0x1F, 0xDD78, Blob.FromHex("A9002003FEA645BD00B18561A9112003FE20B08E8A60"));
 
 			// Check for trapped monster routine, see 11_8EC0_CheckTrap.asm
-			PutInBank(0x11, 0x8EB0, Blob.FromHex("A561202096B030A645BD008FF025856AA9C0203D96A56A200096A903CD866BD00820189668684C43961820E98E201896A2F06020E98E60AA60A911855818A5612093DDA445B90062090499006260"));
+			PutInBank(0x11, 0x8EB0, Blob.FromHex("A561202096B02DA645BD008FF022856AA9C0203D96A56A200096A903CD866BD0062018964C439620E68E201896A2F06020E68E60AA60A911855818A5612093DDA445B90062090499006260"));

```

Comment:
What are the changes? I know this also had some bugfixes recently

Reply:
There's actually a merge error here. I replaced 93DD and lost it while conflict resolving.

Reply:
It now should be the correct blob, with the changed address. Please check the blob, with your version just to make sure.